tests/integration-tests/README.md

```diff
@@ -0,0 +1,532 @@
+# Integration Tests
+
+This directory contains the integration tests for the Open Enterprise Agent (OEA).
+
+## Main concepts
+
+The integration tests are written in Kotlin, and use the following tools and libraries:
+
+1. [Serenity BDD](https://serenity-bdd.github.io/) for test execution engine
+2. [Hoplite](https://github.com/sksamuel/hoplite) for configuration management
+3. [Ktor](https://ktor.io/) for HTTP listener (async receiver for webhook messages)
+4. [PRISM Kotlin client](https://github.com/hyperledger-labs/open-enterprise-agent/packages/1919198) for OEA API models.
+5. [Atala Automation](https://github.com/input-output-hk/atala-automation/) for general testing helpers.
+6. [Awaitility](http://www.awaitility.org/) for asynchronous operations waiting.
+7. [TestContainers](https://www.testcontainers.org/) for Docker containers management.
+
+The project uses Behavior Driven Development (BDD) approach, and the tests are written with the help of Gherkin language.
+The key benefits of the BDD approach are:
+
+* BDD encourages collaboration between developers, testers, and non-technical stakeholders (such as business analysts or product owners).
+* In BDD, requirements and specifications are expressed in natural language, often using a specific format called "Given-When-Then." This format structures the description of the expected behavior of a system in a way that is easy to understand.
+* BDD tools, such as Cucumber and Serenity, help translate these specifications into executable tests. These tests serve as living documentation, ensuring that the software behaves as expected based on the agreed-upon specifications.
+
+The Screenplay pattern is used to write the tests. The pattern is described in detail in the
```

Comment:
We used the Screenplay pattern to write the tests. For more details about the pattern are here